occurrences/schema.py

```diff
@@ -502,15 +502,6 @@ def validate_enrolment(study_group, occurrence, new_enrolment=True):
             raise EnrolmentMaxNeededOccurrenceReached(
                 "Number of enrolled occurrences is greater than the needed occurrences"
             )
-        group_size = (
```

Comment:
This cannot be here anymore because we should allow new enrolment regardless is group size, as long as it's not approved. This validation need to be moved to inside `enrolment.approve`